### ISSUE TITLE
Change FIM fields names to use the correspond alert fields names

### ISSUE THREAD

**Issue by jotacarma90** [wazuh/wazuh#8918]
## Description
Hello team, 
this issue is to work on the names of the FIM fields, which do not correspond to the names of the fields in the alerts, and this causes a lot of confusion and problems. For example, this alert section:

```
"syscheck": {
      "path": "/var/ossec/etc/ossec.conf",
      "size_before": "3663",
      "size_after": "3664",
      "perm_after": "100640",
      "uid_after": "0",
      "gid_after": "999",
      "md5_before": "98b351df146410f174a967d726f9965e",
      "md5_after": "7f4f5846dcaa0013a91bd6d3ac4a1915",
      "sha1_before": "c6368b866a835b15baf20976ae5ea7ea2788a30e",
      "sha1_after": "c959321244bdcec824ff0a32cad6d4f1246f53e9",
      "event": "modified"
    },
```
Shows a 'path' field, but the decoder field name you have to use in a rule, is 'file'. We want the names to be the same, both in the alert and in the field that receives the decoder.
This way it is much easier and intuitive to write rules using these fields.
This is going to generate a compatibility problem with the rules that are already written, so we want to introduce this change in the next version 5.0 of Wazuh.

- Related issue about the documentation that has had to be done to know what the field names are right now, and to which field of the alert they correspond to: https://github.com/wazuh/wazuh-documentation/issues/3251

- Related issue to add a few more fields that can be helpful: https://github.com/wazuh/wazuh/issues/5002

**Comment by chemamartinez** [comment]
## Status

Blocked until the branch 5.0-dev is fixed.

**Comment by vikman90** [comment]
This will be fixed in the new engine: #11334.
